docs/getting-started.rst

```diff
@@ -22,6 +22,8 @@ Installation Quick Reference Guide
 
 1. Get the your board NengoFPGA ready using the
    :ref:`device-specific board setup documentation <board-setup>`.
+#. Once your board is set up, the following steps are performed on the PC connected to
```

Comment:
Do we want to make the following points a sublist (as they are to be done on the PC)?

Or alternatively make "do this on your PC" paragraph text instead of a list item?